packages/alfa-rules/test/common/expectation/get-nodes-between.spec.tsx

```diff
@@ -0,0 +1,131 @@
+import { h } from "@siteimprove/alfa-dom";
+import { Sequence } from "@siteimprove/alfa-sequence";
+import { test } from "@siteimprove/alfa-test";
+import { getNodesBetween } from "../../../src/common/expectation/get-nodes-between";
+
+const hello = h.text("Hello");
+const spanHello = h.element("span", [], [hello]);
+const divHello = <div>{spanHello}</div>;
+
+const spanEmpty = <span></span>;
+const divEmpty = <div>{spanEmpty}</div>;
+
+const world = h.text("World");
+const spanWorld = h.element("span", [], [world]);
+const divWorld = <div>{spanWorld}</div>;
+
+h.document([divHello, divEmpty, divWorld]);
+
+test("getNodesBetween() returns the nodes between two nodes", (t) => {
+  t.deepEqual(
+    getNodesBetween(divHello, divWorld, {
+      includeFirst: true,
+      includeSecond: true,
+    }).toJSON(),
+    Sequence.from([
+      divHello,
+      spanHello,
+      hello,
+      divEmpty,
+      spanEmpty,
+      divWorld,
+    ]).toJSON()
```

Comment:
```suggestion
    getNodesBetween(divHello, divWorld, {
      includeFirst: true,
      includeSecond: true,
    }).toArray(),
    [
      divHello,
      spanHello,
      hello,
      divEmpty,
      spanEmpty,
      divWorld,
    ]
```

Can be done for the remaining cases as well.